bespoken/dashboard
# src: link source fixes

#### Overview
- [x] Fixes linking source.
- [x] Refactors linking source code.
- [x] Removes `createpage/Route.tsx` component:
  - By extracting linking source code to the `Dashboard` component, to keep it more simple and consistent.
  - All the code, can be summarized within `Dashboard.ComponentDidMount`, the interesting part: https://github.com/bespoken/dashboard/compare/linking-source-refactor?expand=1#diff-e0d107de60f5989c4108f2fc3899c3b0R100

#### Test plan
  - Make sure linking source works as expected.
  - Make sure there is not more dependency on `createpage/Route.tsx`, as redirection extra step.
